packages/medusa-plugin-algolia/tsconfig.spec.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
suggestion: should we update the eslintrc and ignore in order to add the lint ability to those package. it is valid for both plugins